app/components/ProgressStepIndicator.tsx

```diff
@@ -69,6 +81,9 @@ export const ProgressStepIndicator: React.FunctionComponent<Props> = ({
         ))}
       </Row>
       <style jsx global>{`
+        h2 {
+          font-size: 1.5rem;
+        }
```

Comment:
move this to a local `style jsx` to avoid side effects (and the ones below if possible)

Reply:
Because we use bootstrap `Row` and `Col` components we have to use `global` here - but there won't be side-effects since this component doesn't render any children

Reply:
ah nevermind, I got confused again with the scope of `global`

Reply:
Wouldn't a global style affect all components on the page where this component is rendered?
Given that the `h2` is rendered directly by this component, you should be able to use a local style